src/components/DefaultProfiles/DefaultProfileDetails.tsx

```diff
@@ -48,72 +130,43 @@ export const DefaultProfileDetails = () => {
   };
 
   return (
-    <div
-      className="main-quick-settings"
-      style={{
-        height: 480,
-        alignItems: 'normal',
-        paddingTop: 100
-      }}
-    >
-      <div
-        style={{
-          width: '100%',
-          paddingLeft: '5px'
-        }}
-      >
-        <div
-          style={{
-            display: 'flex',
-            justifyContent: 'center',
-            alignItems: 'center'
-          }}
-        >
-          <img
+    <MainQuickSettings className="main-quick-settings">
+      <Wrapper>
+        <ImageWrapper>
+          <ProfileImage
             src={`${API_URL}${api.getProfileImageUrl(profile_url)}`}
             alt="No image"
             width="50"
             height="50"
             className="profile-image image-prev"
-            style={{
-              border: `8px solid ${
-                defaultProfile.display?.accentColor ?? '#e0dcd0'
-              }`,
-              display: 'block',
-              position: 'relative'
-            }}
+            borderColor={defaultProfile?.display?.accentColor ?? '#e0dcd0'}
           />
-        </div>
-        <p>{defaultProfile.name}</p>
-        <p
-          style={{
-            padding: '0px 10px',
-            whiteSpace: 'pre-wrap',
-            overflow: 'hidden',
-            scrollBehavior: 'smooth',
-            maxHeight: 283
-          }}
-          ref={mainContainerRef}
-        >
-          {defaultProfile.display?.description}
-          <div
-            className={`settings-item active-setting`}
-            style={{
-              marginTop: 80,
-              marginBottom: 80
-            }}
-          >
-            <div
-              className="settings-entry"
-              style={{
-                padding: '6px'
-              }}
-            >
+        </ImageWrapper>
+        <Name>{defaultProfile?.name}</Name>
+        <Description ref={mainContainerRef}>
+          {defaultProfile?.display?.description ||
+            'Shared by a member of the community, this profile is ready for you to discover in the cup. Perfect for those who enjoy exploring new extractions and finding their own balance.'}
```

Comment:
Note to myself: I should create a GitHub action to ensure all profiles to be properly describes